widgetssdk/src/main/java/com/glia/widgets/chat/ChatContract.kt

```diff
@@ -57,6 +57,7 @@ internal interface ChatContract {
         fun onTakePhotoClicked()
         fun onImageCaptured(result: Boolean)
         fun onContentChosen(uri: Uri)
+        fun onLocalImageItemClick(attachment: LocalAttachment, view: android.view.View)
```

Comment:
'Local' word for me has a bit of broad meaning, which required me to second gues what this method does. Could it be 'downloaded' or 'saved' instead?

Reply:
This is actually not downloaded or saved but a `local` file picked by the visitor.